gradlew.bat

```diff
@@ -0,0 +1,89 @@
+@rem
+@rem Copyright 2015 the original author or authors.

```

Comment:
Is this copyright ok?

Reply:
I copied this exec from Etendo Core. Should I update the copyright info?

Reply:
Raised this question to the knowledge area. Copyright info should be left unchanged